### PR TITLE
Add tests for exact in/out and swap exits

### PR DESCRIPTION
**Pull request by agualis**
# Description

- New unit tests for _exact-in_ exit handler.  
- New unit tests for _exact-out_ exit handler.  
- New unit tests for swap exit handler.  
- New helper to create random addresses for tests. 
- New join/exit builders to simplify testing setup.
- New test token address constants to improve test readability. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [x] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/approvals/useTokenApprovalActions.spec.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ test('Shows correct labels', async () => {
   );
 });
 
-const spender = '0x8c9e6c40d3402480ACE624730524fACC5482798c';
+const spender = randomAddress();
```
Comment (garethfuller) [inline src/composables/approvals/useTokenApprovalActions.spec.ts:31]: Spender will typically be the Vault address or a relayer address, so I'm not sure if setting it to a random address makes sense here. But I guess it also doesn't matter.

File: src/services/balancer/pools/exits/handlers/exact-out-exit.handler.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const gasPriceServiceMock: DeepMockProxy<GasPriceService> =
+  mockDeep<GasPriceService>();
+
+async function mountExactOutExitHandler(pool: Pool) {
+  return new ExactOutExitHandler(ref(pool), getBalancer(), gasPriceServiceMock);
+}
+
+const exitParams = buildExitParams({
+  bptIn: '0.00000000000000001',
```
Comment (garethfuller) [inline src/services/balancer/pools/exits/handlers/exact-out-exit.handler.spec.ts:28]: For exact out handler it's the `amountsOut` that are critical in constructing the tx, bptIn is irrelevant.

File: src/services/balancer/pools/exits/handlers/swap-exit.handler.spec.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+const gasPriceServiceMock: DeepMockProxy<GasPriceService> =
+  mockDeep<GasPriceService>();
+
+async function mountSwapExitHandler(pool: Pool) {
+  return new SwapExitHandler(ref(pool), getBalancer(), gasPriceServiceMock);
+}
+
+const exitParams = buildExitParams({
+  bptIn: '0.00000000000000001',
```
Comment (garethfuller) [inline src/services/balancer/pools/exits/handlers/swap-exit.handler.spec.ts:29]: For swap exits there are two modes:
1. OutGivenIn - where `bptIn` is the relevant input
2. InGivenOut - where `amountsOut` is the the relevant input
Maybe we should have a test for both.

**Review by garethfuller — APPROVED** [review]

File: src/services/balancer/pools/exits/handlers/exact-out-exit.handler.spec.ts (hunk 2 above)
Comment (agualis) [reply src/services/balancer/pools/exits/handlers/exact-out-exit.handler.spec.ts:28]: Thanks. I will improve this tests with this feedback in a future PR.

**Pull request merged**